packages/migrate/uglifyJsPlugin/__snapshots__/uglifyJsPlugin.test.js.snap

```diff
@@ -78,7 +78,7 @@ exports[`uglifyJsPlugin transforms correctly using "uglifyJsPlugin-3" data 1`] =
 `;
 
 exports[`uglifyJsPlugin transforms correctly using "uglifyJsPlugin-4" data 1`] = `
-"const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+"const TerserPlugin = require('terser-webpack-plugin');
```

Comment:
Interesting, why not all snapshots were updated? Just these 2?

Reply:
@PlayMa256 Good question! This is the only one that failed during test run